### PR TITLE
Bump to ADNLPModels 0.5

### PR DESCRIPTION
**Pull request by tmigot**
In particular, the print of models is fixed with this release.
Test #25

### CODE REVIEW

**Comment by tmigot** [comment]
We actually need to modify the MATRIX https://github.com/JuliaSmoothOptimizers/JSOTutorials.jl/blob/2cdce897c57d284e29c65ff59b6a1a877bec0c81/.github/workflows/Deploy-website.yml#L27 too so that the .jmd file corresponding to the .toml file is added to the list of scripts.

File: tutorials/introduction-to-adnlpmodels/Project.toml [hunk 1]
```diff
@@ -4,5 +4,5 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
 
 [compat]
-ADNLPModels = "0.4"
+ADNLPModels = "0.4, 0.5"
```
Comment (abelsiqueira) [inline tutorials/introduction-to-adnlpmodels/Project.toml:7]:
```suggestion
ADNLPModels = "0.5"
```
I prefer to use stronger bounds, so there is no doubt that it is using the correct version

**Review by abelsiqueira — CHANGES_REQUESTED** [review]
Does this need anything else?

**Comment by tmigot** [comment]
Finally, no more ugly display!

**Pull request merged**